.travis.yml

```diff
@@ -1,19 +1,61 @@
 language: rust
-rust:
-  - stable
-sudo: false
+rust: stable
 
 cache: cargo
 
-addons:
-  apt:
-    packages:
-      - libusb-1.0-0-dev
-
+matrix:
+  include:
+    - name: Linux with packaged libusb-1.0 library
+      os: linux
+      addons:
+        apt:
+          packages:
+            - libusb-1.0-0-dev
+      env:
+        - PKGCONFIG: libusb-1.0 libudev
```

Comment:
I noticed that the real PKGCONFIG set only libusb-1.0.
https://travis-ci.com/a1ien/rusb/jobs/270644961#L187

Reply:
Fixed.